addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1528,13 +1530,16 @@ const Wysiwyg = Widget.extend({
                         }
                         this.odooEditor.historyPauseSteps();
                         try {
-                            this._processAndApplyColor(eventName, ev.data.color);
+                            this._processAndApplyColor(eventName, ev.data.color, true);
                         } finally {
                             this.odooEditor.historyUnpauseSteps();
                         }
                     });
                     colorpicker.on('color_leave', null, ev => {
-                        this.odooEditor.historyRevertCurrentStep();
+                        if (!this._colorPicked) {
```

Comment:
I'm confused. This looks like a bugfix ?

Reply:
Uh yeah that seems about right, I'll take it into a separate commit then.